articles/quickstart/webapp/any/01-login.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to integrate Auth0 authentication into any conventional server-based web application.
+budicon: 
+---
+
+This guide explains in general terms how to integrate Auth0 authentication into a conventional server-based web app serving static HTML pages to the user. 
+
+## Overview
+
+This is how user authentiation happens between a conventional server-side web application and the Auth0 service. 
+
+1. Your application redirects the user's browser to Auth0 `/authorize` endpoint, with a *redirect URI* in the query string.
+2. The user logs in, using the Auth0 login form on that page.
+3. Auth0 authenticates the user. 
+    - Authentication is performed against the list of users provided by you to Auth0.
```

Comment:
is not that the list is provided by the developer to Auth0, there's a user database, that might have been created in different ways